### PR TITLE
Add harshakp06 to remote India learners

### PR DESCRIPTION
**Pull request by harshakp06**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**: 
This PR is for verification of my GitHub account for the  contribution to Kubernetes project 

Fixes #

**Special notes for your reviewer**:
added the harshakp06.md file to the India/remote/learners folder
6 
**Does this PR introduce a user-facing change?**:
This PR won't fix anything 
went with the slides/doc provided by the Kubernetes project for the sign of Contributor License Agreement (CLA) 

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by nikhita** [comment]
@harshakp06 Welcome! Please update the commit message to something like `Add harshakp06 to remote India learners` .

**Comment by nikhita** [comment]
Please also change the commit message, not just the PR title. Thanks!

**Comment by harshakp06** [comment]
@nikhita 
can you check once - does it work

**Comment by harshakp06** [comment]
Changed commit message to **Add harshakp06 to remote India learners**

File: remote/learners/jaswir.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hello K8S!
+Jaswir Raghoe here from the Netherlands.
```
Comment (nikhita) [inline remote/learners/jaswir.md:2]: Please remove changes to this file

**Comment by harshakp06** [comment]
Hi @nikhita 
I had updated the commit message with changes

File: remote/learners/jaswir.md (hunk 1 above)
Comment (harshakp06) [reply remote/learners/jaswir.md]: I  had updated the commit message with changes

**Review by nikhita — APPROVED** [review]
Thank you for your patience in getting this merged 😄 

/lgtm
/approve

/woof

**Pull request merged**